src/services/beers.js

```diff
@@ -8,14 +8,13 @@ export async function fetchRandomBeer() {
     const { id, image_url: img, name, description, ibu, abv } = beer;
     return ({ id, img, name, description, ibu, abv })
   } catch (error) {
-    console.log(error)
```

Comment:
Leave error logs until you don't have some error handling approach

Reply:
Done!